### PR TITLE
Responsive design for login page

### PR DESCRIPTION
**Pull request by liannivin**

### CODE REVIEW

File: public/assets/css/pages/index.css [hunk 1]
```diff
@@ -1,71 +1,76 @@
...
-  }
+	from {
+		background-position: 0 0;
+	}
 
-  to {
-    background-position: -1200px 600px;
-  }
+	to {
+		background-position: -1200px 600px;
```
Comment (cdsandoval) [inline public/assets/css/pages/index.css]: Well job doing this. What is this -1200px doing ?

File: public/assets/css/pages/index.css [hunk 2]
```diff
@@ -1,71 +1,76 @@
...
+	text-align: center;
+	padding: 0.2em 0.4em;
+	margin-bottom: 25px;
+	border-radius: 5px;
+	border: 3px solid #000;
+}
+
+@media screen and (min-width: 320px) {
+	.window.login {
+		height: 100vh;
```
Comment (cdsandoval) [inline public/assets/css/pages/index.css]: Woah using vh is great as units. Well done.

File: public/assets/css/pages/index.css (hunk 1 above)
Comment (liannivin) [reply public/assets/css/pages/index.css]: @yummta

File: public/assets/css/pages/index.css (hunk 2 above)
Comment (liannivin) [reply public/assets/css/pages/index.css]: Thanks. Using media queries we have to use only relative units of measurement.

**Pull request merged**